### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by supssaha**

### CODE REVIEW

**Comment by supssaha** [comment]
url bug fix

File: lib/jiraConnect.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (smartameer) [inline lib/jiraConnect.js:2]: Unwanted lines

File: lib/jiraConnect.js [hunk 2]
```diff
@@ -20,6 +22,7 @@
                     'Authorization' : this.config.authorization
                 }
             };
+            console.log(endpoint);
```
Comment (smartameer) [inline lib/jiraConnect.js:25]: Unwanted log statements

File: lib/jiraConnect.js [hunk 3]
```diff
@@ -179,6 +182,30 @@
                     'Authorization' : this.config.authorization
                 }
             };
+console.log(endpoint);
```
Comment (smartameer) [inline lib/jiraConnect.js:185]: Unwanted console statement

File: lib/jiraConnect.js [hunk 4]
```diff
@@ -179,6 +182,30 @@
...
+                if (!error && response.statusCode === 200) {
+                    cb(null, body, config);
+                } else {
+                    cb({error: error, module:"jira:fetchProjectSprints"}, null, config);
+                }
+            }
+        });
+    };
+
+jiraConnect.prototype.fetchProjectSprintsbyrapidview = function fetchProjectSprintsbyrapidview (config, callback) {
```
Comment (smartameer) [inline lib/jiraConnect.js:197]: Not properly indented

File: lib/jiraConnect.js [hunk 5]
```diff
@@ -179,6 +182,30 @@
...
+        var cb = callback || null,
+           endpoint = config.url,
+            options = {
+                url: endpoint,
+                json: true,
+                headers: {
+                    'Authorization' : this.config.authorization
+                }
+            };
+console.log(endpoint);
```
Comment (smartameer) [inline lib/jiraConnect.js:208]: Unwanted console statement

File: lib/jiraConnect.js [hunk 6]
```diff
@@ -197,14 +224,16 @@
         console.log("Connection Started for fetching rapidview "+config.rapidviewid+"'s sprintid "+ config.sprintid);
         var cb = callback || null,
             endpoint = this.config.urilist.sprint.replace(/{rapidviewid}/g, config.rapidviewid)
-                .replace(/{sprintid}/g, config.sprintid),
+             .replace(/{sprintid}/g, config.sprintid),
```
Comment (smartameer) [inline lib/jiraConnect.js:227]: whats the change?

File: lib/jiraConnect.js [hunk 7]
```diff
@@ -197,14 +224,16 @@
...
+          
             options = {
                 url: endpoint,
                 json: true,
                 headers: {
                     'Authorization' : this.config.authorization
                 }
             };
+            console.log(endpoint);
```
Comment (smartameer) [inline lib/jiraConnect.js:236]: Unwanted console statement

File: lib/jiraConnect.js [hunk 8]
```diff
@@ -215,6 +244,32 @@
             }
         });
     };
+
+jiraConnect.prototype.fetchProjectRapidviews = function fetchProjectRapidviews (config, callback) {
```
Comment (smartameer) [inline lib/jiraConnect.js:248]: where in method dtsatement rapidview id is being used?

Comment (smartameer) [reply lib/jiraConnect.js:248]: Not indented properly

File: lib/jiraConfig.js [hunk 9]
```diff
@@ -14,7 +14,7 @@
                 this.authorization = 'Basic ' + btoa( options.authorization.username + ':' + options.authorization.password);
             }
             this.ssl = this.resturi.match(/^https/) !== null ? true : false;
-            var path = "/jira/rest";
+            var path = "/rest";
```
Comment (smartameer) [inline lib/jiraConfig.js:17]: Default is `/jira/rest`, if needed we can configure it through arguments for path configuration.

**Pull request closed without merging**